docs/content/preview/develop/quality-of-service/transaction-priority.md

```diff
@@ -157,3 +162,85 @@ yugabyte=> select * from account;
   </tr>
 
 </table>
+
+### Show transaction priority types
+
+The following is another example which demonstrates the usage of `yb_transaction_priority` based on the priority types.
+
+1. From an active [ysqlsh](../../../admin/ysqlsh/#starting-ysqlsh) shell, create a table as follows:
+
+   ```sql
+   CREATE TABLE test_scan (i int, j int);
+   ```
+
+1. Start by setting the lower and upper bound values for your transaction.
```

Comment:
The numbering seems off. It should be 2., 3., and so on.

Reply:
@lnguyen-yugabyte , thats the numbering standard with markdown. If you look at this PR's build preview: https://deploy-preview-13445--infallible-bardeen-164bc9.netlify.app/preview/develop/quality-of-service/transaction-priority/, it will render as a numbered list.